pystorm/bolt.py

```diff
@@ -50,6 +58,7 @@ class Bolt(Component):
         from pystorm.bolt import Bolt
 
         class SentenceSplitterBolt(Bolt):
+            output_fields = ['word']
```

Comment:
I'm not a fan of adding all sorts of meta magic for this, but I'm not sure we have much of a choice.